src/client/pages/Parkeren/useParkerenData.hook.tsx

```diff
@@ -42,14 +42,17 @@ export function useParkerenData() {
     : VERGUNNINGEN;
   const vergunningen = vergunningenState.content;
 
-  const parkeervergunningen = getFilteredVergunningen(vergunningen);
+  const decosParkeerVergunningen = getFilteredVergunningen(vergunningen);
+  const hasMijnParkerenVergunningen = !!PARKEREN.content?.isKnown;
 
   return {
     tableConfig,
-    parkeervergunningen,
+    decosParkeerVergunningen,
+    hasMijnParkerenVergunningen,
     isLoading: isLoading(vergunningenState),
     isError: isError(vergunningenState),
-    parkerenUrlSSO: PARKEREN.content?.url,
+    parkerenUrlSSO:
+      PARKEREN.content?.url ?? import.meta.env.REACT_APP_SSO_URL_PARKEREN,
```

Comment:
Ik denk eigenlijk dat er altijd een `url` waarde uit zou moeten komen.

Reply:
Typescript gaat hier dus flippen omdat het nog kan dat content undefined is (ook al stop je die url erin in de backend wanneer parkeren niks heeft). Wat ik wel snap want de backend kan een error terug geven. En in dat geval, hoe onwaarschijnlijk ook hebben we de url wel.

We kunnen dubbel die url definieren, maar dat lijkt me ook vervelend tenzij we kunnen refereren naar een backend variable in de infra. Dan lijkt het me wel okay.

Reply:
Alright deze is en de comment hieronder is ook veranderd. Alles komt nu vanuit de API.